forge-gui/res/cardsfolder/upcoming/great_unclean_one.txt

```diff
@@ -0,0 +1,12 @@
+Name:Great Unclean One
+ManaCost:4 B
+Types:Creature Demon
+PT:4/5
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigLoseLife | TriggerDescription$ Reverberating Summons — At the beginning of your end step, each opponent loses 2 life. Then for each opponent who has less life than you, create a 1/3 black Demon creature token named Plaguebearer of Nurgle.
+SVar:TrigLoseLife:DB$ LoseLife | Defined$ Opponent | LifeAmount$ 2 | SubAbility$ RepeatOpps
+SVar:RepeatOpps:DB$ RepeatEach | RepeatPlayers$ Player.Opponent | RepeatSubAbility$ DBToken
+SVar:DBToken:DB$ Token | TokenScript$ plaguebearer_of_nurgle | ConditionCheckSVar$ LifeOpponent | ConditionSVarCompare$ LELifeYou | TokenOwner$ You 
+SVar:LifeOpponent:PlayerCountRemembered$LifeTotal
```

Comment:
Try using this for TokenAmount to avoid the repeating:
`SVar:NumLessLifeOpp:PlayerCountOpponents$HasPropertylifeLTCount$YourLifeTotal`